### PR TITLE
Add 404 Page

### PR DESCRIPTION
**Pull request by ahmedsilat44**
## Related Issue

- Created a 404 page

Closes: #236 

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

| Original screenshot 
image
 | Updated screenshot 
image
 

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

<a href="https://gitpod.io/#https://github.com/WebXDAO/WebXDAO.github.io/pull/239"></a>

### CODE REVIEW

File: src/components/Global/Navbar.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const Navbar = () => {
             <div className="flex items-center justify-between w-full md:w-auto">
               <Link href="/">
                 <>
-                 <a href="https://webxdao.github.io/">
+                 <a href="/">
```
Comment (mkubdev) [inline src/components/Global/Navbar.js:28]:
```suggestion
                 <a>
```
In fact we don't even need the a href, because we use nextjs/Link :)

**Review by mkubdev — COMMENTED** [review]
LGTM, kindly suggest you to remove the href in the a tag before merging. ⚡

**Comment by ahmedsilat44** [comment]
> LGTM, kindly suggest you to remove the href in the a tag before merging. ⚡

Oh done.

File: src/components/Global/Navbar.js (hunk 1 above)
Comment (ahmedsilat44) [reply src/components/Global/Navbar.js]: @mkubdev This was added because of the bug. making the logo not working

**Review by mkubdev — APPROVED** [review]

**Review by vinzvinci — APPROVED** [review]
LGTM!

**Pull request merged**